erpnext/accounts/report/general_ledger/general_ledger.py

```diff
@@ -193,6 +193,11 @@ def get_data_with_opening_closing(filters, account_details, gl_entries):
 	# closing
 	data.append(totals.closing)
 
+	#total closing
+	total_closing = totals.total_closing
+	total_closing['debit'] = totals.closing.get('debit', 0) - totals.closing.get('credit', 0)
```

Comment:
It should be set for `debit_in_account_currency` as well

Reply:
Fixed